### ISSUE TITLE
Placenta: Maternal vascular malperfusion

### ISSUE THREAD

**Issue by janine-pappas** [obophenotype/human-phenotype-ontology#6579]
**Preferred term label: **Maternal vascular malperfusion**
*Synonyms **Maternal vascular underperfusion**
**Definition (free text, please give PubMed ID) recognizable pattern of placental injury related to altered uterine and intervillous blood flow 
PMID: **27223167**
https://pubmed.ncbi.nlm.nih.gov/27223167/
Parent term (use hpo.jax.org/app)  **HP:0100767 Abnormal Placental Morphology**  
Your nano-attribution (ORCID)**  **https://orcid.org/0000-0002-9579-3626**

**Comment by pnrobinson** [comment]
adding, thanks @saintfrison
